bin/build

```diff
@@ -4,18 +4,21 @@ set -e
 CURRENT_DIR=$(pwd)
 
 if [[ ! $INPUT_DIR ]] || [[ ! $OUTPUT_DIR ]]; then
-  echo "Input/Output directory variables must be set.\n"
+  printf "Input/Output directory variables must be set.\n"
   exit 1
 fi
 
 mkdir -p $INPUT_DIR
 mkdir -p $OUTPUT_DIR
 
 if [[ ! -d $INPUT_DIR ]] || [[ ! -d $OUTPUT_DIR ]]; then
-  echo "Input/Output directories must exist.\n"
+  printf "Input/Output directories must exist.\n"
   exit 1
 fi
 
+# Copy Meteor distribution to $HOME
+cp -R /opt/home /mnt
```

Comment:
in the `DNX` image we do: `cp -R /opt/home/. $HOME`, but you only need `.meteor`, right?